tdrs-backend/tdpservice/security/test/test_owasp_zap_scan.py

```diff
@@ -0,0 +1,54 @@
+"""Unit tests for OWASP ZAP scan operations."""
+from django.contrib.admin.models import LogEntry
+import pytest
+
+from tdpservice.security.models import OwaspZapScan
+from tdpservice.users.models import User
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'owasp_zap_scan__fail_count, '
+    'owasp_zap_scan__pass_count, '
+    'owasp_zap_scan__warn_count, '
+    'expected_result',
+    [
+        (5, 95, 0, 'Failed'),
+        (0, 95, 5, 'Warning'),
+        (0, 100, 0, 'Passed'),
+        (0, 0, 0, 'Error')
+    ]
+)
+def test_owasp_zap_scan_result(owasp_zap_scan, expected_result):
```

Comment:
pytest-factoryboy auto creates the `owasp_zap_scan` fixture from the register call in `conftest.py`